src/custom-elements/src/components/button/button.tsx

```diff
@@ -48,6 +48,11 @@ export class Button {
    */
   @Prop({ reflect: true }) center = false;
 
+  /**
+   *  to describe button for screen readers.
+   */
+  @Prop({ reflect: true }) description: string;
```

Comment:
@janibolkvadze Let's change the attribute name from `description` to `alternative-text`